opentelemetry-python-core

```diff
@@ -0,0 +1 @@
+../opentelemetry-python
```

Comment:
Is this file expected as part of this change?

Reply:
No, this is not expected. This was supposed to be a draft PR. I probably created a regular PR by accident. Will fix in the morning.

Reply:
Removed it.